etna/datasets/tsdataset.py

```diff
@@ -140,6 +160,24 @@ def make_future(self, future_steps: int) -> "TSDataset":
         Returns
         -------
         dataset with features in the future.
+
+        Examples
+        --------
+        >>> from etna.datasets import generate_const_df
+        >>> df = generate_const_df(
+        ...    periods=30, start_time="2021-06-01",
+        ...    n_segments=2, scale=1
+        ... )
+        >>> df_ts_format = TSDataset.to_dataset(df)
+        >>> ts = TSDataset(df_ts_format, "D")
+        >>> ts.make_future(4)
+        segment    segment_0 segment_1
```

Comment:
could you please add example with regressors?

Reply:
I've added regressors to current example